Sources/Quick/NSString+C99ExtendedIdentifier.swift

```diff
@@ -22,12 +21,25 @@ public extension NSString {
         return invalidCharacters
     }()
 
-    @objc(qck_c99ExtendedIdentifier)
```

Comment:
Because there's a workaround, I think we should preserve this public API.

Reply:
I'm okay to preserve this for 1.x but I'd like to hide this in 2.x, because I assume the reason why this API is public is to expose it to the generated header. We can now expose internal Swift APIs to the generated header thanks to #791, so there will be no need to make it public.

Reply:
Can we explicitly deprecate it?

Reply:
I tried adding `@available(*, deprecated)` but `__attribute__ ((deprecated))` is not exposed to the generated header unfortunately.